apps/passport-server/.env.example

```diff
@@ -1,70 +1,217 @@
+# by setting the following environment variable to 'true'.
+# If GENERIC_RATE_LIMIT_DISABLED='true' then *all* rate-limiting is disabled,
+# irrespective of whether ACCOUNT_RESET_RATE_LIMIT_DISABLED is set.
+ACCOUNT_RESET_RATE_LIMIT_DISABLED='true'
+
+# Set this to an integer value between 1 and the number of
+# CPU cores you have available. The server offloads CPU-bound
+# tasks to these workers
+WORKER_QUANTITY=1
+
+##################################################################
+##################################################################
+#   Frogcrypto
+##################################################################
+##################################################################
+
+# A stringified JSON array of email addresses whose Zupass user can manage FrogCrypto via UI and api routes
+# e.g. '["test@example.com"]'
+# FROGCRYPTO_ADMIN_USER_EMAILS=
+
+# The base URL for frogcrypto assets
+# FROGCRYPTO_ASSETS_URL=
+
+##################################################################
+##################################################################
+#   Podbox.dev (localhost:3002) (client on localhost:3005)
+##################################################################
+##################################################################
+
+# So that Podbox server knows the url of where its corresponding client lives.
+# 
+GENERIC_ISSUANCE_CLIENT_URL="http://localhost:3005"
+
+# To enable server-issued PCDs using the generic issuance feature, the server needs an an EdDSA 
+# private key. You can generate one using the following command at the root of the project, 
+# after installing dependencies and building the project:
+# 
+# node -e 'console.log(require("@pcd/eddsa-pcd").newEdDSAPrivateKey())' | pbcopy
+GENERIC_ISSUANCE_EDDSA_PRIVATE_KEY="129806f9d68c3cf87450a968748fd31f01ba4e6f2078d80f24ebab23d98df7cf"
+
+# To recognise email PCD credentials, generic issuance needs to know the Zupass public key.
+# https://api.zupass.org/issue/eddsa-public-key as quoted JSON
+# e.g. '["233423ad", "ab454545"]' (except with much longer strings!)
+#
+# node -e 'require("@pcd/eddsa-pcd").getEdDSAPublicKey("PRIVATE KEY").then(k => console.log(JSON.stringify(k))).catch(console.log)' | pbcopy
+GENERIC_ISSUANCE_ZUPASS_PUBLIC_KEY=["00f669040a1c31ff18b8e221b94ac36580da68a05c69c21298569e97e193ca45","2b2a9ae6ed7d5ca397637bbe7180849ac6e828171790644ed723abf2decb96c2"]
+
+# for ease of use in local development, lets you specify a user to be an admin
+# in the generic issuance service
+# e.g. you could set the value to be ["ivan@0xparc.org"] to make the server
+# set the user with that email address to be an admin on start-up.
+# 
+GENERIC_ISSUANCE_ADMINS=["ivan@0xparc.org"]
+
+# if true, and both `STYTCH_PROJECT_ID` and `STYTCH_SECRET` are
+# not set, lets anyone log in as any user just by providing an
+# email address. this feature is disabled in prod.
+STYTCH_BYPASS=true
```

Comment:
@rrrliu - this environment variable toggles dev-mode login for the server, but you'll also need to make a corresponding change in the generic issuance `.env` file, which i'll describe in a comment there